app/utils/validation.js

```diff
@@ -48,16 +48,21 @@ export const sameAs = (otherField, message) => (value, context) => [
   message,
 ];
 
-export function createValidator(fieldValidators) {
+export function createValidator(fieldValidators, rawValidator) {
+
   return function validate(input) {
-    return Object.keys(fieldValidators).reduce((errors, field) => {
+  const resultRawValidator = rawValidator(input)
+
+    const validatorErrors = Object.keys(fieldValidators).reduce((errors, field) => {
```

Comment:
...Then call this fieldValidatorErrors, to show that these errors are separate from the rawValidatorErroa